lib/ui/page/home/widget/block.dart

```diff
@@ -82,6 +83,10 @@ class Block extends StatelessWidget {
   /// Maximum width this [Block] should occupy.
   final double maxWidth;
 
+  /// Whether to clip overflowing content in height, but not width.
+  /// Defaults to `false` to avoid extra GPU work.
+  final bool clipHeight;
```

Comment:
This documentation doesn't follow Effective Dart Documentation guidelines. https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph